engine/pkg/cmd/cli/cli_job.go

```diff
@@ -61,12 +61,13 @@ func (o *jobGeneralOptions) addFlags(cmd *cobra.Command) {
 // validate checks that the provided job options are valid.
 func (o *jobGeneralOptions) validate(ctx context.Context, cmd *cobra.Command) error {
 	if len(o.masterAddrs) == 0 {
-		return errors.ErrInvalidCliParameter.GenWithStack("master-addrs can't be nil")
+		o.masterAddrs = []string{"127.0.0.1:10240"}
+		log.Warn("the master-addrs are not assigned, use default addr: 127.0.0.1:10240")
```

Comment:
IMO we can add a global variable with default value, and use it as the default value in command flags.

Reply:
PTAL